masonry/src/testing/harness.rs

```diff
@@ -526,7 +540,7 @@ impl TestHarness {
             return;
         }
 
-        let new_image = self.render();
+        let new_image: DynamicImage = self.render().into();
 
```

Comment:
I'm not sure. I'm wanting to redo a lot of this testing code, but I think that will want to come after linebender/rfcs#7

Reply:
Wait, no, I misread the code. We already redraw in the early exit, the only thing this skips is actually sending the Vello scene to GPU. So that makes sense.